Makefile

```diff
@@ -22,9 +22,7 @@ ALL_SRC := $(shell find . -name "*.go" | grep -v -e vendor \
 TEST_TIMEOUT := "-timeout=3s"
 
 .PHONY: test
-test: examples
-	$(ECHO_V)go test $(RACE) $(TEST_TIMEOUT) $(PKGS)
```

Comment:
This test fails on race condition which is skipped: https://github.com/uber-go/fx/blob/master/service/host_test.go#L275